contentScript.js

```diff
@@ -1,3 +1,30 @@
+let clickedElement;
+let interval;
+
+document.addEventListener('mousedown', event =>{
+    //right click
+    if(event.button == 2) {
+        clickedElement = event.target;
+        const onBlur = () => {
+            clearInterval(interval);
+            clickedElement.removeEventListener('blur', onBlur);
+            clickedElement = null;
+        }
+        clickedElement.addEventListener('blur', onBlur);
+    }
+}, true);
+
+chrome.runtime.onMessage.addListener(request => {
+    if (request === 'contextMenuClicked') {
+        writeOnElement(clickedElement)
+    }
+});
+
+/*
+    Move this code to a file called type.js
+    To do this we need to start using webpack since ES6 modules is not supported
+*/
```

Comment:
what if we leave in this file forever 😅, I don't think this code will grow that much.

Reply:
Maybe.